csireverseproxy/deploy/config.yaml

```diff
@@ -1,19 +1,7 @@
-mode: Linked
+mode: StandAlone
```

Comment:
@boyamurthy : do we need the mode parameter itself?

Reply:
Yes, this is needed.